chef/cookbooks/provisioner/templates/default/autoyast.xml.erb

```diff
@@ -27,6 +27,13 @@
     </signature-handling>
     <storage/>
   </general>
+  <bootloader>
```

Comment:
Meaning, the XML tags remain the same for SLE11 and SLE12. I do not see any use for if/else statement here.